samples/clients/cpp-console/RaspberryPi.md

```diff
@@ -24,15 +24,28 @@ There are many ways to do development on a Raspberry pi. It may be useful to tak
 
 ## Setting up the code
 
-The repo should be cloned onto your device and we will operate out of the cpp-console folder
+* Clone the Voice Assistant git repo onto your device
 
-Make sure you have the Microsoft speech SDK downloaded. Links are in the main readme
+  ```sh
+  git clone https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant.git
+  ```  
 
-For Linux or ARM devices the target version and the native binaries should be copied to the lib folder in this repo so that you have a structure like this (for ARM32): ./lib/arm32. 
+* Download the Speech SDK
 
-Headers and their folders should be copied into the include folder so that you have a structure like this: ./include/cxx_api and ./include/c_api
+  ```sh
```

Comment:
This is good for now, but I was thinking we could put it in the build script or create a script that does this for the user instead of making them call all the individual commands

Reply:
I agree, I decided to take baby steps and get something simple initially. Also the part I hate is that the actual commands would vary based on the Speech SDK version, so will look at having a script to handle that.